### PR TITLE
Add complete RWD python requirements

### PR DESCRIPTION
**Pull request by caseypt**
Adds all of the Python requirements necessary to run RWD.

Part of https://github.com/WikiWatershed/docker-rwd/pull/1

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1,7 @@
 nose==1.3.4
+Fiona==1.5.1
+numpy==1.10.1
+pandas==0.17.1
+pyshp==1.2.3
+scipy==0.16.1
+Shapely==1.2.19
```
Comment (kdeloach) [inline requirements.txt]: We may be able to target the latest versions of these libraries since GDAL and Proj4 are built from source now.

**Comment by kdeloach** [comment]
+1

**Comment by caseypt** [comment]
I think I may have to do something like [this](https://gist.github.com/dan-blanchard/7045057) to get this to build on Travis.

File: requirements.txt (hunk 1 above)
Comment (caseypt) [inline requirements.txt]: Good point. I'll update and see if it works.

**Comment by emiliom** [comment]
FYI, if you are using or intend to use conda packaging, you should take a look at our [anaconda.org IOOS channel](https://anaconda.org/ioos).

All the dependencies/requirements listed for RWD are already handled as up-to-date conda packages on this channel.

For more info and usage details, see:
- https://github.com/ioos/conda-recipes/wiki
- https://github.com/ioos/conda-recipes

Pinging @aufdenkampe, only b/c he and I have talked extensively about conda and the IOOS channel.

**Comment by caseypt** [comment]
Thanks @emiliom. I'll check it out.

I'm going to merge this to make it easier to update and review https://github.com/WikiWatershed/docker-rwd/pull/1.

**Pull request merged**